src/Voron/Data/CompactTrees/PersistentDictionary.Generated.cs

```diff
@@ -0,0 +1,10 @@
+
+/// DO NOT MODIFY.
+/// All changes should be done through the usage of the Voron.Dictionary.Generator
+
+namespace Voron.Data.CompactTrees;
+partial class PersistentDictionary
+{
+    private const int NumberOfPagesForDictionary = 8;
+    public const int MaxDictionarySize = 1439;
```

Comment:
In what units is this value?

Reply:
In dictionary entries, gonna update it to reflect that.  Also, will check but probably there is no need for it to be public, a lower visibility could do.